### PR TITLE
Created a pull request template

### LINKED ISSUE #15: Add a pull request template ?
Not sure about this one

### PR DESCRIPTION
**Pull request by Mews**
- [x] Closes #15  

## Changes
Created a simple pull request template.\
I tried to have it reflect the guidelines present in [CONTRIBUTING.md](https://github.com/Seluj78/flask-utils/blob/main/CONTRIBUTING.md).

## Summary by CodeRabbit

- **Documentation**
  - Introduced a new pull request template to standardize descriptions, related issues, and checklists for contributors.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<!--- If applicable, please type an `x` in one of the boxes below and provide the issue number after the # sign: -->
+- [ ] Closes #
+- [ ] Related to #
+
+## Checklist
+<!--- Make sure you have completed the following steps before submitting your PR -->
+<!--- Feel free to type an `x` in all the boxes below to let us know you have completed the steps: -->
+- [ ] Checked that the pre-commit checks pass (see [your-first-code-contribution](https://github.com/Mews/flask-utils/blob/main/CONTRIBUTING.md#your-first-code-contribution) and also [pre-commit](https://docs.pymc.io/en/latest/contributing/python_style.html))
+- [ ] Checked that all the tests pass (see [your-first-code-contribution](https://github.com/Mews/flask-utils/blob/main/CONTRIBUTING.md#your-first-code-contribution))
+- [ ] Checked that your commit messages follow [the specified guidelines](https://github.com/Mews/flask-utils/blob/main/CONTRIBUTING.md#commit-messages)
```
Comment (Seluj78) [inline .github/pull_request_template.md:18]:
```suggestion
- [ ] Checked that the pre-commit checks pass (see [your-first-code-contribution](https://github.com/Seluj78/flask-utils/blob/main/CONTRIBUTING.md#your-first-code-contribution) and also [pre-commit](https://docs.pymc.io/en/latest/contributing/python_style.html))
- [ ] Checked that all the tests pass (see [your-first-code-contribution](https://github.com/Seluj78/flask-utils/blob/main/CONTRIBUTING.md#your-first-code-contribution))
- [ ] Checked that your commit messages follow [the specified guidelines](https://github.com/Seluj78/flask-utils/blob/main/CONTRIBUTING.md#commit-messages)
```

Comment (Mews) [reply .github/pull_request_template.md]: Oops I didn't realize I put the links from my fork

**Review by Seluj78 — APPROVED** [review]
🙌 LGTM

**Comment by Seluj78** [comment]
Thank you very much for the PR @Mews !

**Pull request merged**